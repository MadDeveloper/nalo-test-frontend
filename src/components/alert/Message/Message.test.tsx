import { BrowserRouter } from "react-router-dom"
import renderer from "react-test-renderer"
import { Message } from "."

describe("Components > Message", () => {
  it("should match snapshot", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Message>Lorem ipsum</Message>
          <Message error>Lorem ipsum</Message>
          <Message warning>Lorem ipsum</Message>
          <Message info>Lorem ipsum</Message>
          <Message success>Lorem ipsum</Message>
          <Message action={<span>Action</span>}>Lorem ipsum</Message>
          <Message link="/" linkLabel="Back home">
            Lorem ipsum
          </Message>
        </BrowserRouter>
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
