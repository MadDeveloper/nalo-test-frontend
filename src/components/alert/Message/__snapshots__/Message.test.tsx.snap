// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > Message should match snapshot 1`] = `
Array [
  <div
    className="root"
  >
    <div
      className="content"
    >
      <span
        className="root"
      >
        Lorem ipsum
      </span>
    </div>
  </div>,
  <div
    className="root error"
  >
    <div
      className="content"
    >
      <span
        className="root"
      >
        Lorem ipsum
      </span>
    </div>
  </div>,
  <div
    className="root warning"
  >
    <div
      className="content"
    >
      <span
        className="root"
      >
        Lorem ipsum
      </span>
    </div>
  </div>,
  <div
    className="root info"
  >
    <div
      className="content"
    >
      <span
        className="root"
      >
        Lorem ipsum
      </span>
    </div>
  </div>,
  <div
    className="root success"
  >
    <div
      className="content"
    >
      <span
        className="root"
      >
        Lorem ipsum
      </span>
    </div>
  </div>,
  <div
    className="root"
  >
    <div
      className="content"
    >
      <span
        className="root"
      >
        Lorem ipsum
      </span>
    </div>
    <div
      className="actionIconContainer"
    >
      <span>
        Action
      </span>
    </div>
  </div>,
  <div
    className="root"
  >
    <div
      className="content"
    >
      <span
        className="root"
      >
        Lorem ipsum
        <span
          className="root"
        >
           
          <a
            className="link"
            href="/"
            onClick={[Function]}
          >
            Back home
          </a>
        </span>
      </span>
    </div>
  </div>,
]
`;
